pkg/util/proxyconfig/merge.go

```diff
@@ -23,6 +23,8 @@ func MergeUserSystemNoProxy(proxy *configv1.Proxy, infra *configv1.Infrastructur
 	set := sets.NewString(
 		"127.0.0.1",
 		"localhost",
+		".svc",
+		".svc.cluster.local",
```

Comment:
did you confirm these with @sdodson ?

Reply:
```suggestion
		".cluster.local",
```
We had `.cluster.local` rather than `.svc.cluster.local`
https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/openshift_facts/library/openshift_facts.py#L852-L863

Reply:
@bparees @sdodson I updated the PR to use `.cluster.local` instead of  `.svc.cluster.local`.